src/domain/services/query.service.ts

```diff
@@ -85,16 +86,23 @@ export class QueryService {
     return whereLog;
   }
 
+  obsoleteWhere(body: any) {
+    const obsolete = !!body?.obsolete;
+    delete body?.obsolete;
+    return `${obsolete ? '' : 'not '}pt.obsolete`;
+  }
+
   async count(body: any) {
     const start = Date.now();
     this.logger.debug(body);
 
+    const obseleteWhere = this.obsoleteWhere(body);
```

Comment:
```suggestion
    const obsoleteWhere = this.obsoleteWhere(body);
```

Reply:
eagle eye :wink:

Reply:
I really have trouble spelling that word!